tfexec/cmd.go

```diff
@@ -74,6 +79,14 @@ func (tf *Terraform) buildEnv(mergeEnv map[string]string) []string {
 		env[checkpointDisableEnvVar] = os.Getenv(checkpointDisableEnvVar)
 	}
 
+	// always override user agent
+	ua := mergeUserAgent(
+		os.Getenv(appendUserAgentEnvVar),
+		tf.appendUserAgent,
+		fmt.Sprintf("HashiCorp-terraform-exec/%s", version.ModuleVersion()),
```

Comment:
Shouldn't these values be in reverse order? i.e. static `HashiCorp-terraform-exec` first and then other values being _appended_, rather than prepended?

Reply:
I think it depends. arguably "terraform cloud" is more significant / distinct than terraform exec I think in the case of that system? As there will be many terraform exec apps, so the app itself is the more significant identifier?

Reply:
I don't disagree, but the problem is that the ENV variable is called _append_, not _prepend_, so we would need to reflect it there somehow (with another/changed variable), else the behaviour won't match the name.

Reply:
Thats the common convention in programming but not the english definition of the word. It was a poor original choice for the env var, but its what we have so no option to change it now.